components/shared/ViewSidebar.vue

```diff
@@ -79,92 +75,65 @@ watch(
-    <AlertsIntroPanel
-      v-if="showIntroPanel && alertsStatistics"
-      :calculate-hectares="calculateHectares"
-      :date-options="dateOptions"
-      :data-for-alerts-intro-panel="dataForAlertsIntroPanel"
-      :logo-url="logoUrl"
-      :show-slider="showSlider"
-      :alerts-statistics="alertsStatistics"
-      @date-range-changed="emit('date-range-changed', $event)"
-    />
-    <DataFeature
-      v-if="feature"
-      :allowed-file-extensions="allowedFileExtensions"
-      :feature="filteredFeature"
-      :file-paths="filePaths"
-      :is-alert="isAlert"
-      :media-base-path="mediaBasePath"
-      :media-base-path-alerts="mediaBasePathAlerts"
-    />
-    <DownloadMapData
-      v-if="downloadAlert"
-      :data-for-download="localAlertsData"
-    />
+  <div
+    class="fixed top-0 left-0 h-full w-[400px] bg-white shadow-lg transform transition-transform duration-300 ease-in-out overflow-y-auto z-50"
+    :class="{ 'translate-x-0': showSidebar, '-translate-x-full': !showSidebar }"
+  >
+    <div class="relative h-full">
+      <div v-if="!scrolled" class="scroll-indicator">
+        <ChevronDown class="w-6 h-6 text-gray-600 animate-bounce" />
+      </div>
+      <button
+        class="absolute top-4 right-4 p-2.5 bg-white/80 backdrop-blur-sm hover:bg-gray-100 rounded-full transition-all duration-200 ease-in-out shadow-sm hover:shadow-md active:scale-95 focus:outline-none focus:ring-2 focus:ring-gray-200 focus:ring-offset-2"
+        @click="emit('close')"
+      >
+        <span class="sr-only">Close</span>
+        <X class="w-5 h-5 text-gray-600" />
+      </button>
+
+      <div class="p-4">
+        <AlertsIntroPanel
+          v-if="showIntroPanel && alertsStatistics"
+          :calculate-hectares="calculateHectares"
+          :date-options="dateOptions"
+          :data-for-alerts-intro-panel="dataForAlertsIntroPanel"
+          :logo-url="logoUrl"
+          :show-slider="showSlider"
+          :alerts-statistics="alertsStatistics"
+          @date-range-changed="emit('date-range-changed', $event)"
+        />
+        <DataFeature
+          v-if="feature"
+          :allowed-file-extensions="allowedFileExtensions"
+          :feature="filteredFeature"
+          :file-paths="filePaths"
+          :is-alert="isAlert"
+          :media-base-path="mediaBasePath"
+          :media-base-path-alerts="mediaBasePathAlerts"
+        />
+        <DownloadMapData
+          v-if="downloadAlert"
+          :data-for-download="localAlertsData"
+        />
+      </div>
+    </div>
   </div>
 </template>
 
 <style scoped>
-.sidebar {
-  position: fixed;
-  top: 0;
-  left: 0;
-  height: 100%;
-  width: 400px;
-  background: white;
-  padding: 20px;
-  box-shadow: 2px 0 5px rgba(0, 0, 0, 0.1);
-  overflow-y: auto;
-  z-index: 1000;
-}
-
-@media (max-width: 768px) {
-  .sidebar {
-    height: 50%;
-    width: 100%;
-    bottom: 0;
-    top: auto;
-  }
-
-  .scroll-indicator {
-    display: block !important;
-  }
-}
```

Comment:
I like the work you've done here, but I'd like to propose an additional change. Before, we only showed `scroll-indicator` for smaller viewports. Now, we are showing it for any viewport, which is fine: but, I think we should only show this icon _if_ the sidebar is scrollable. Right now, the icon shows even when that is not the case:

![image](https://github.com/user-attachments/assets/9cd55387-c886-4cb6-ab2a-cc62e707fc3d)